ostree-compliance-mode.spec

```diff
@@ -0,0 +1,37 @@
+Name:           ostree-compliance-mode
+Version:        0.1
+Release:        1%{?dist}
+Summary:        ostree-compliance-mode
+
+License:        GPLv2
+URL:            https://github.com/ericcurtin/%{name}
+Source0:        %{url}/archive/main.tar.gz
```

Comment:
This should be similar to twincam, I made a real release here:

https://github.com/ericcurtin/ostree-compliance-mode/archive/refs/tags/0.1.tar.gz

Reply:
LGTM